install/installer/pkg/components/slowserver/render_test.go

```diff
@@ -60,6 +60,27 @@ func TestServerDeployment_UsesToxiproxyDbHost(t *testing.T) {
 	}
 }
 
+func TestServerDeployment_DbWaiterUsesToxiproxyDbHost(t *testing.T) {
+	ctx := renderContext(t, nil, true)
+
+	objects, err := deployment(ctx)
+	require.NoError(t, err)
+
+	require.Len(t, objects, 1, "must render only one object")
+
+	deployment := objects[0].(*appsv1.Deployment)
+
+	for _, c := range deployment.Spec.Template.Spec.InitContainers {
+		if c.Name == "database-waiter" {
+			for _, e := range c.Env {
+				if e.Name == "DB_HOST" {
+					require.Equal(t, toxiproxy.Component, e.Value)
+				}
+			}
+		}
+	}
+}
```

Comment:
This will pass if there's no init-contianer with name= waiter, is that desirable?

If not, you could filter the init containers for the waiter and assert it has 1 entry

Reply:
Thanks, I've added the filtering and check for one expected container.